KDIC/BlogTableCell.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+class BlogTableCell: UITableViewCell {
+    
+  
+    @IBOutlet weak var blogPostTitle: UILabel!
```

Comment:
As before, no need to prefix these with `blog`, and you're naming the `label` not the actual text.